backend/tests/v5/e2e/routes/teamspaces/projects/federations/federations.test.js

```diff
@@ -38,27 +38,55 @@ const project = {
 	name: ServiceHelper.generateRandomString(),
 };
 
+const modelWithRevId = ServiceHelper.generateUUIDString();
+const modelWithoutRevId = ServiceHelper.generateUUIDString();
+
 const modelSettings = [
 	{
 		_id: ServiceHelper.generateUUIDString(),
 		name: ServiceHelper.generateRandomString(),
 		isFavourite: true,
 		properties: { ...ServiceHelper.generateRandomModelProperties(), federate: true },
+		subModels: [{ model: modelWithRevId }],
```

Comment:
don't you want subModels inside properties? otherwise it won't go into the settings?